### PR TITLE
[967093][Screen Readers- CosmosDB – Notification] Screen reader does not pass the combo-box list information

### PR DESCRIPTION
**Pull request by Chris-MS-896**
Repro steps:
Open URL: https://ms.portal.azure.com/ 
Login via valid credentials
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate to Expand/Collapsed button in the bottom and select it.
Navigate through all the controls like combo box, clear notifications.
Verify, whether screen reader pass the list item information or not.
Actual Result:
Screen reader does not pass the combo-box list information under notification field. User is able to navigate to the list items however screen reader does not pass any information about it.

Expected Result:
Screen reader should pass the list items present under combo-box with their position so that user can easily navigate to the control present under it.

Current solution: change dropdownlist component to component under lib of office-ui-fabric-react

### CODE REVIEW

**Review by zfoster — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -6359,7 +6359,6 @@
           "version": "5.7.1",
```
Comment (zfoster) [inline package-lock.json:6359]: why did this change?

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import * as React from "react";
 import { ClientDefaults, KeyCodes } from "../../../Common/Constants";
 import AnimateHeight from "react-animate-height";
-
+import { Dropdown, IDropdownOption } from "office-ui-fabric-react/";
```
Comment (southpolesteve) [inline src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:8]:
``` suggestion
import { Dropdown, IDropdownOption } from "office-ui-fabric-react";
```

**Review by southpolesteve — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (Chris-MS-896) [reply package-lock.json]: Thanks, I solved this

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx (hunk 2 above)
Comment (Chris-MS-896) [reply src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx]: Sure, I changed

**Pull request merged**